app.go

```diff
@@ -116,6 +116,12 @@ type App struct {
 	latestGroup *Group
 	// TLS handler
 	tlsHandler *TLSHandler
+	// check added routes of sub-apps
+	subAppsRoutesAdded bool
```

Comment:
property is used in one place and set directly from false to true without actually being used anywhere else 

do we really need this property ?

Reply:
app.Test executes buildTree in each usage. We shouldn't mount app again in tests

Reply:
Okay make sense
Why not using sync.Once instead of the boolean property?